### ISSUE TITLE
calculation on anharmonicity of amorphous using DynaPhopy

### ISSUE THREAD

**Issue by KunHeeYe** [abelcarreras/DynaPhoPy#15]
Dear Abel Carreras:

Hello, I have a question related to vibrational free energy calculated by DynaPhopy.

I want to calculate the vibrational free energy of amorphous by considering anharmonicity, so I'm performing VASP MD in various temperatures. The values of quasi-particle and power spectrum free energy are quite similar to the harmonic free energy. Thus, the anharmonic free energy value does not seem unreasonable.
However, can DynaPhopy be applied to the non-crystalline system like the amorphous phase? 

Kun Hee Ye

**Comment by abelcarreras** [comment]
I guess you can apply as long as you have a super-cell that describes well the amorphous material.
Have you checked the DOS? if it is reasonable I would say it is OK. Which force constants are you using
to get the eigenvectors to do the projection?

**Comment by KunHeeYe** [comment]
Thank you for your fast reply!
I simulated amorphous HfO2, whose cell size is about 10 Å (96 atoms).
The DOS seems reasonable for me except for a slight imaginary frequency near 0 THz. 
I'm using the force constant of 1x1x1 supercell amorphous (about 10 Å) calculated by phonopy.  I tried to increase the cell size, but it was not easy to expand the cell size because the symmetry is low.

By the way, I have other questions related to anharmonic free energy.
1. What is the difference between quasi-particle and power spectrum free energy? The two values ​​are generally similar, but I'm still wondering which one to use. My goal is to get accurate amorphous free energy.
2. Generally, the anharmonicity will increase as temperature increases. Therefore, the difference between harmonic and anharmonic free energy will increase as the temperature goes up. However, in my case, the difference is larger at 900K than at 1000K. Is it caused by inaccurate or insufficient MD simulation? I used POTIM=2 and NSW = 80000.
3. Finally, is it ok to use NVE ensemble? In the case of the NVT ensemble, the temperature vibrates, so I'm using the NVE ensemble that can accurately control the temperature through SMASS=-1.
